monai/bundle/scripts.py

```diff
@@ -1969,7 +1968,7 @@ def create_workflow(
         workflow_ = workflow_class(**_args)
 
     workflow_.initialize()
-
+    _log_input_summary(tag="run", args=_args)
```

Comment:
move it here then the logging summary can be logged by filehandler used in the bundle.